### ISSUE TITLE
batch_size and the unstable loss

### ISSUE THREAD

**Issue by SkySeaWalk** [lzx551402/tfmatch#5]
@lzx551402 @zjhthu Hi, thanks a lot for your great work.
I'v trained aslfeat with circle loss , and when I use tensorboard to see the loss , I found it decreased unstably from step 5K to 400K. I also check the default batch_size is 2.
1) Did you do some experiments on the batch_size or other hyperparameters? 
2) Would the loss be more stable, if I increase the batch_size? or get a better model ?
3) which base model did you use, when you train dcn with circle loss ? 380K step, 100K, or others? 

Best Regards! look forward to your reply.

**Comment by EstherJinyan** [comment]
I have the same problem with you,have you solved this problem?

**Comment by EstherJinyan** [comment]
> @lzx551402 @zjhthu Hi, thanks a lot for your great work.
> I'v trained aslfeat with circle loss , and when I use tensorboard to see the loss , I found it decreased unstably from step 5K to 400K. I also check the default batch_size is 2.
> 
> 1. Did you do some experiments on the batch_size or other hyperparameters?
> 2. Would the loss be more stable, if I increase the batch_size? or get a better model ?
> 3. which base model did you use, when you train dcn with circle loss ? 380K step, 100K, or others?
> 
> Best Regards! look forward to your reply.

I have the same problem with you,have you solved this problem?
